analytics/terraform/spark-eks-ipv6/addons.tf

```diff
@@ -103,7 +103,7 @@ module "eks_blueprints_addons" {
     }
   }
   karpenter = {
-    chart_version       = "0.37.0"
+    chart_version       = "1.0.0"
```

Comment:
You might need to uplift data addons tag and update the Karpenter resources to v1.

Reply:
good call. I was pulling v1.34.0 but we should set that as the min. 

When i rolled this out the existing karpenter resource configs seemed to get created as v1 but i'm still working through the configs to see if they need updates to the spec.